src/registerServiceWorker.js

```diff
@@ -9,100 +9,96 @@
 // This link also includes instructions on opting out of this behavior.
 
 const isLocalhost = Boolean(
```

Comment:
I've seen this being done before, I don't think you have to cast it to a Boolean, since it's JS it should work straight away.

Reply:
Noted. Will pass on the suggestion to person who worked on the feature. @NardusRetroRabbit